news/2017-08-15-osu-mania-4k-world-cup-group-stage-recap.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "osu!mania 4K World Cup - Group Stage Recap"
+date: 2017-08-15 14:37:30 +0000
+---
+
+Hi everyone, mangomizer here! Summer may be ending, but the excitement continues! I’m here today of course, to catch up on the latest events regarding the hugely anticipated osu!mania 4K World Cup!
+
+![](/wiki/shared/news/banners/mania4k_logo_2017.jpg)
+
+This past weekend saw the conclusion of the Group Stage, where we had 184 talented players representing 32 countries squaring off against each other to qualify for the Round of 16. If you’ve missed the weekend matches, you can always rewatch the VODs [here](https://www.twitch.tv/osulive/videos/all), or settle for the weekly recap below! 
+
+This year’s Group Stage saw a grand total of 46 matches packed tightly into the span of 2 days. Courtesy of our beloved mappool selectors, we were treated to a truly diverse mappool featuring artists from all around the world. As expected, the mappool difficulty has increased substantially from last year, though this has not discouraged players in the slightest. 
```

Comment:
"this" double dutches with previous paragraph, sub for 2017's group stage or something